### PR TITLE
Use new ACX codelab repo and live example URL

### LINKED ISSUE #1001: ACX codelab: ensure that autodeploy works as it does for other examples
- [x] Dart-doc-syncer updates https://github.com/angular/dart-doc-syncer/pull/34
- [ ] local build script updates, #1044, #1045

### PR DESCRIPTION
**Pull request by chalin**
Fixes #1001 

Staged at https://angulardart-org-dev3.firebaseapp.com/codelabs/angular_components

With this PR, the ACX codelab app versions are published on webdev (along with all other AngularDart doc examples).  Links are given in each codelab page.

### CODE REVIEW

File: src/_codelabs/angular_components/2-easystart.md [hunk 1]
```diff
@@ -16,13 +16,13 @@ In this step, you’ll change the app to use a few of the AngularDart Components
 *   \<glyph>
 *   \<acx-scorecard>
 
+{% include_relative _run_example.md %}
```
Comment (kwalrath) [inline src/_codelabs/angular_components/2-easystart.md]: These links didn't work for me. The page name is `2-easystart`, and the branch is `2-starteasy`.

File: src/_codelabs/angular_components/3-usebuttons.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ using these components:
 These controls appear in two custom components: \<lottery-simulator>
 and \<settings-component>.
 
+{% include_relative _run_example.md %}
```
Comment (kwalrath) [inline src/_codelabs/angular_components/3-usebuttons.md]: These links DID work for me.

Comment (kwalrath) [reply src/_codelabs/angular_components/3-usebuttons.md]: When I went to the [next page](https://angulardart-org-dev3.firebaseapp.com/codelabs/angular_components/4-final), I was surprised not to have run/view links there, like we did on pages 2 & 3. We should be consistent, one way or another.

File: src/_codelabs/angular_components/1-base.md [hunk 3]
```diff
@@ -11,40 +11,16 @@ prevpage:
...
 In this step, you’ll download the code for the app, and you’ll familiarize yourself with both the app and the components you’ll be adding to it.
 
+{% include_relative _run_example.md %}
 
 ## <i class="fa fa-money"> </i> Get the app code
 
-Download the app’s code from the
-[dart-lang/one-hour-codelab](https://github.com/dart-lang/one-hour-codelab)
-GitHub repo
+Download all of the code for this codeelab from the
```
[the remark this answers is not in the document]
Comment (chalin) [reply src/_codelabs/angular_components/1-base.md]: Fixed

File: src/_codelabs/angular_components/1-base.md [hunk 4]
```diff
@@ -11,40 +11,16 @@ prevpage:
...
-{% endprettify %}
-
-<p> This creates a directory named <code>one-hour-codelab</code>. </p>
-</li>
-</ul>
-
-All the code for this codelab is under the **`angular_components`**
-directory of `one-hour-codelab` or `one-hour-codelab-{{site.branch}}`.
-
+{%- endif %} using the "Clone or download" dropdown.
```
[the remark this answers is not in the document]
Comment (chalin) [reply src/_codelabs/angular_components/1-base.md]: Changed, and thanks for the reference!

[the remark this answers is not in the document]
Comment (chalin) [reply src/_codelabs/angular_components/1-base.md]: Done

File: src/_codelabs/angular_components/2-easystart.md (hunk 1 above)
Comment (chalin) [reply src/_codelabs/angular_components/2-easystart.md]: Ah, good catch! I've renamed the page to match the folder name.

File: src/_codelabs/angular_components/3-usebuttons.md (hunk 2 above)
Comment (chalin) [reply src/_codelabs/angular_components/3-usebuttons.md]: I've added it to the 4-final page.

**Pull request merged**